galera/Project1/Developer.cpp

```diff
@@ -3,9 +3,43 @@
 
 using namespace std;
 
-void Developer::setLanguage(string language)
+
+void Developer::setLanguage(language  language)
 {
```

Comment:
Next time implement each issue on a separate branch. Changes made to Developer class are not related to the issue with setName, so it shouldn't be here. Please, keep that in mind for future